### DISCUSSION TITLE
Date format on Excel export

### DISCUSSION THREAD

**Discussion by 2BCAD** [kimai/kimai#2473] [Questions & Answers]
Hi.
I switched to Kimai V2 and I found that when I do an export of my times, the date format is different from the one in V1.

In V2 the date is shown as YYYY-mm-dd where in V1 it was dd-mm-YYYY, which I prefere since I am in Europe.

Is there a way to change the format of the date when exporting data to Excel?

Thanks in advance.

Bob

**Comment by kevinpapst** [comment]
Change the language in your user preferences.

**Reply by 2BCAD** [reply]
Thanks for the reply.
The language for my user is set to german.  That should give me the wished output, no?

![Bild_2021-03-31_152958](https://user-images.githubusercontent.com/20596817/113152000-f407b580-9235-11eb-9e6f-9aabc41d4621.png)

**Reply by kevinpapst** [reply]
I guess so, but if your Excel is english, dates will probably be reformatted. Try CSV.

**Reply by 2BCAD** [reply]
I did export the data to csv. When I open it with a text editor, the date format is still the same.
"2021-03-04","08:00","08:30"

**Reply by kevinpapst** [reply]
You are right. I can't remember if that was on purpose or if it was just never changed to use the user language.

For now: select the column and format the cells as date time in your wished format in Excel.

**Reply by 2BCAD** [reply]
Thanks.
That's what I do. I just wanted to know if there was maybe setting that I missed.

Vielen Dank!

**Reply by kevinpapst** [reply]
Sorry for the bad news 😁 I believed myself that the format changes, but the code says otherwise.

I will see what I can do about it. I will work on the export in the next time anyways, so it can be configured. Then the date format can be one of the new settings.
